matrix-sdk-android/src/main/java/org/matrix/android/sdk/internal/SessionManager.kt

```diff
@@ -51,6 +51,13 @@ internal class SessionManager @Inject constructor(private val matrixComponent: M
         }
     }
 
+    fun stopSession(sessionId: String) {
+        if (sessionComponents.containsKey(sessionId).not()) {
+            throw RuntimeException("You don't have a session for id $sessionId")
+        }
+        sessionComponents[sessionId]!!.session().stopSync()
```

Comment:
To avoid usage of `!!`, maybe store `sessionComponents[sessionId]` in a val and test for nullability?

Reply:
good point :+1:

Reply:
done 9c1d6e0484c33d6a471790a9017db57954babb40